{"name": "test_user_login_wrong", "status": "passed", "attachments": [{"name": "stdout", "source": "399f6c91-8237-42aa-bb35-07ed48b8a0e8-attachment.txt", "type": "text/plain"}], "start": 1600238085771, "stop": 1600238085975, "uuid": "c38b01fa-fe9c-49e5-bdcf-1edf5877875b", "historyId": "8923e36b93bfcbf2b1b29bba9c76fef6", "testCaseId": "4b6bcd6bc9f4b93fe085c2939884ea45", "fullName": "testcase.test_login.TestLogin#test_user_login_wrong", "labels": [{"name": "severity", "value": "critical"}, {"name": "feature", "value": "测试登录模块"}, {"name": "story", "value": "测试登录密码错误"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "TX5C3S1L7D31UYD"}, {"name": "thread", "value": "14912-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_login"}], "links": [{"type": "test_case", "url": "/basic/auth/login", "name": "/basic/auth/login"}]}